docs/02-execution-context.md

```diff
@@ -24,7 +24,7 @@ Contains shared state from hooks.
 
 ## `t.passed`
 
-Whether a test has passed. This value is only accurate in the `test.afterEach()` and `test.afterEach.always()` hooks.
+Whether a test has passed. This value is only accurate during test execution (including teardown hooks) and in the `test.afterEach()` and `test.afterEach.always()` hooks.
```

Comment:
```suggestion
When used in `test.afterEach()` or `test.afterEach.always()` hooks this tells you whether the test has passed. When used in a test itself (including teardown functions) this remains `true` until an assertion fails, the test has ended with an error, or a teardown function caused an error. This value has no meaning in other hooks.
```

Reply:
To be honest, now when I have more or less good understanding of how AVA works (using it quite extensively), this looks to me obvious without additional clarifications. 
Probably (can't really imagine the case) for newcomers that might be useful somehow. 
But understanding the fact that test execution stops on the first assertion error, is enough. 
But most likely I just don't have unbiased view on this.

Reply:
> But understanding the fact that test execution stops on the first assertion error, is enough.

It doesn't though. AVA's assertions don't throw errors so execution won't stop.

Reply:
Hmm.. that's new. 
Probably I was confused by not having other assertion results for all failed assertions.
Then it's already beneficial :)
It's a topic for another ticket/conversation, but is there a sense to continue test execution if the results of that execution are unknown anyway?

Reply:
This has been discussed a fair bit elsewhere — apologies for not looking up the issue numbers 👍